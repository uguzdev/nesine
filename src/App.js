import React, { useState } from 'react';
import "./App.scss";

function App() {
    const [text, setText] = useState('Nesine Frontend Case');

    return <div>{text}</div>
}

export default App;
