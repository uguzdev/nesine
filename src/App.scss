@import 'style/globals';

body {
  background: $nesine-yellow;
  margin: 0;
}
