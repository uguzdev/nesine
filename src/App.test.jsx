import React from 'react';
import { render, screen } from '@testing-library/react';
import renderer from 'react-test-renderer';
import App from './App';

test('renders learn react link', () => {
  render(<App />);
  const linkElement = screen.getByText(/nesine/i);
  expect(linkElement).toBeInTheDocument();
});

test('Snapshot', () => {
  const tree = renderer.create(<App />).toJSON();
  expect(tree).toMatchSnapshot();
});
