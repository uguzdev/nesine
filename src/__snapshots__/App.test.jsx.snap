// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot 1`] = `
<div>
  Nesine Frontend Case
</div>
`;
